sci-physics/geant-vmc/geant-vmc-9999.ebuild

```diff
@@ -9,25 +9,49 @@ if [[ ${PV} == *9999* ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://github.com/vmc-project/geant4_vmc.git"
```

Comment:
Please add `KEYWORDS=""` below this line.

Reply:
Thanks for pointing this out! Fixed :-).